l10n_br_account/models/account_move_line.py

```diff
@@ -105,6 +105,12 @@ class AccountMoveLine(models.Model):
         string="PIS ST CST Code",
     )
 
+    partner_is_public_entity = fields.Boolean(related="partner_id.is_public_entity")
+
+    allow_csll_irpj = fields.Boolean(
+        compute="_compute_allow_csll_irpj",
+    )
+
```

Comment:
esses 2 campos não vem pelo _inherits do `l10n_br_fiscal.document.line` que por sua vez herda dos campos do `l10n_br_fiscal/models/document_fiscal_line_mixin.py` ?

Reply:
É quando é herdado pelo inherits o compute do campo não tá sendo chamado, no sale order que o mixim é herdado diretamente o problema não acontece, mas na fatura da ruim :/ aí fui obrigado a repetir a definição do campo no account

Reply:
entendi. Ai vale um comentário no código senão ninguem mais vai lembrar depois. E o partner_is_public_entity, da ruim no related dele se não repetir?

Reply:
é que na verdades isso já tá sendo feito para um grupo de campos, se for ver um pouco mais acima, na linha 75 está o comentário:

    # The following fields belong to the fiscal document line mixin
    # but they are redefined here to ensure they are recomputed in the
    # account.move.line views.

Reply:
saquei valeu!